### ISSUE TITLE
migrate:fresh > Command "db:wipe" is not defined

### ISSUE THREAD

**Issue by Kristories** [laravel-zero/laravel-zero#252]
If I run `php <application> migrate:fresh`, I got error :

```
Command "db:wipe" is not defined.  
                                     
  Did you mean this?                 
      db:seed            
```

illuminate/database//Console/Migrations/FreshCommand.php

```
/**
 * Execute the console command.
 *
 * @return void
 */
public function handle()
{
    if (! $this->confirmToProceed()) {
        return;
    }

    $database = $this->input->getOption('database');

    $this->call('db:wipe', array_filter([
        '--database' => $database,
        '--drop-views' => $this->option('drop-views'),
        '--drop-types' => $this->option('drop-types'),
        '--force' => true,
    ]));

    $this->call('migrate', array_filter([
        '--database' => $database,
        '--path' => $this->input->getOption('path'),
        '--realpath' => $this->input->getOption('realpath'),
        '--force' => true,
        '--step' => $this->option('step'),
    ]));

    if ($this->needsSeeding()) {
        $this->runSeeder($database);
    }
}
```

**Comment by nunomaduro** [comment]
What is your version of Laravel Zero?

**Comment by Kristories** [comment]
@nunomaduro  `v2.3.0`

**Comment by pxgamer** [comment]
This was resolved in laravel-zero/framework#367, so we just need to release a v6.1.0 of Laravel Zero.

**Comment by Kristories** [comment]
Thank you @pxgamer @nunomaduro
